python/paddle/distributed/auto_parallel/parallelizer_v2.py

```diff
@@ -80,13 +88,6 @@ def parallel(self, rank):
             self._logger.info(
                 "within parallel partitioner time: {}, mode {}".format(
                     time.time() - time0, self._mode))
-            # Generate optimizer
```

Comment:
Should not move the location of optimizer before recompute and amp pass accept completed program.

Reply:
Done.